packages/llamaindex/package.json

```diff
@@ -20,12 +20,7 @@
     "llamaindex"
   ],
   "dependencies": {
-    "@anthropic-ai/sdk": "0.32.1",
-    "@aws-crypto/sha256-js": "^5.2.0",
-    "@aws-sdk/client-sso-oidc": "^3.693.0",
     "@discoveryjs/json-ext": "^0.6.1",
```

Comment:
remove unused packages: `gpt-tokenizer` and `js-tiktoken` (are in `env`) and `@discoveryjs/json-ext` (is in readers)